src/cards/polymorph.ts

```diff
@@ -0,0 +1,194 @@
+import { CardCategory, UnitType } from '../types/commonTypes';
+import { playDefaultSpellSFX } from './cardUtils';
+import { Spell, addTarget, allModifiers, getCurrentTargets, refundLastSpell } from './index';
+import { CardRarity, probabilityMap } from '../types/commonTypes';
+import * as Unit from '../entity/Unit';
+import * as Pickup from '../entity/Pickup';
+import * as Image from '../graphics/Image';
+import { allUnits } from '../entity/units';
+import { isModActive } from '../registerMod';
+import Underworld from '../Underworld';
+import { IPickup, isPickup, pickups } from '../entity/Pickup';
+import { bossmasonUnitId } from '../entity/units/deathmason';
+import floatingText from '../graphics/FloatingText';
+import seedrandom from 'seedrandom';
+import { chooseObjectWithProbability, chooseOneOf, chooseOneOfSeeded, getUniqueSeedString } from '../jmath/rand';
+
+export const polymorphId = 'Polymorph';
+const spell: Spell = {
+  card: {
+    id: polymorphId,
+    category: CardCategory.Curses,
+    supportQuantity: true,
+    manaCost: 40,
```

Comment:
This should probably be more expensive, maybe 60 or 80.  What do you think?

Reply:
Personal opinion? Unpredictable effects feel inherently weak to me in Spellmasons. Why would I risk making a level more difficult when I can consistently guarantee a win using more controlled/surefire tactics? Feels like the only use case right now is deleting summoners and min-maxing capture soul once a level is under control.

That being said, I haven't used the spell much in real games, and it is quite good against summoners, so I bumped it up to 60.